R/modules.R

```diff
@@ -9,9 +9,7 @@
 #'
 #' @param ...
 #' - For `modules()`: (`teal_module` or `teal_modules`) see [module()] and [modules()] for more details.
-#' - For `toString()`: (optional) additional parameters to pass to recursive calls of `toString`.
-#' - For `print()`: parameters passed to `toString`.
-#'
+#' - For `format()` and `print()`: (optional) additional parameters to pass to recursive calls of `format`.
```

Comment:
A common phrase in R documentation is 
`... arguments passed to other methods`

Reply:
Yeah I know, but for `format` it actually passes an argument (recursively) to `format` and not anything else : ) For `print` it only passes `...` to `format`, So I think the phrasing is accurate.

Reply:
Doesn't `format.modules` also pass to `format.module`?

Reply:
changing to `... arguments passed to other methods
`

Reply:
bf7fd6b16a2fdb4d7f15084fe7017ebdf3ca148b